quickcheck-dynamic/src/Test/QuickCheck/StateModel.hs

```diff
@@ -409,6 +379,57 @@ instance forall state. StateModel state => Arbitrary (Actions state) where
               | otherwise = ps : map (p :) (go ps)
          in go acts
 
+newtype MoreActions state = MoreActions {getMoreActions :: Actions state}
```

Comment:
We need some documentation as this type is now part of the interface of q-d: how to use it, what does it provide, ...